docs/references/api/cors.rst

```diff
@@ -1,7 +1,18 @@
+.. _cors:
+
 CORS
 ====
 
-PostgREST sets highly permissive cross origin resource sharing, that is why it accepts Ajax requests from any domain.
+PostgREST can set any cross origin resource sharing, that is why it can accept Ajax requests from any set of domains.
+
+You can specify the allowed CORS origins using the :code:`server-cors-allowed-origins` config. If the config is not set, it accepts requests from any domain. If the config is set to :code:`""`, it rejects requests from any domain. See :ref:`server_cors_allowed_origins`.
```

Comment:
These parts would fit better on the config page.

```
If the config is not set, it accepts requests from any domain. If the config is set to :code:`""`, it rejects requests from any domain.
```

They can be bullets like here https://postgrest--688.org.readthedocs.build/en/688/references/configuration.html#server-host